src/WebJobs.Script.WebHost/Diagnostics/FunctionsDiagnosticLogsEventSource.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
+{
+    [EventSource(Guid = "B7AB9421-18D8-4C0A-8389-7CB78944D4A4")]
+    public sealed class FunctionsDiagnosticLogsEventSource : ExtendedEventSource
+    {
+        internal static readonly FunctionsDiagnosticLogsEventSource Instance = new FunctionsDiagnosticLogsEventSource();
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1030:UseEventsWhereAppropriate")]
+        [Event(65525, Level = EventLevel.Verbose, Channel = EventChannel.Operational, Version = 1)]
+        public void RaiseFunctionsDiagnosticEventVerbose(string resourceId, string operationName, string category, string location, string monitorLevel, string properties)
+        {
+            if (IsEnabled())
+            {
+                WriteEvent(65525, resourceId, operationName, category, location, monitorLevel, properties);
+            }
+        }
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1030:UseEventsWhereAppropriate")]
+        [Event(65526, Level = EventLevel.Informational, Channel = EventChannel.Operational, Version = 1)]
+        public void RaiseFunctionsDiagnosticEventInformational(string resourceId, string operationName, string category, string location, string monitorLevel, string properties)
+        {
+            if (IsEnabled())
+            {
+                WriteEvent(65526, resourceId, operationName, category, location, monitorLevel, properties);
+            }
+        }
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1030:UseEventsWhereAppropriate")]
+        [Event(65527, Level = EventLevel.Warning, Channel = EventChannel.Operational, Version = 1)]
+        public void RaiseFunctionsDiagnosticEventWarning(string resourceId, string operationName, string category, string location, string monitorLevel, string properties)
```

Comment:
Convert arguments names to PascalCase. Rest of the tables in MDS we use column names with PascalCase.